superset/views/utils.py

```diff
@@ -261,9 +261,8 @@ def get_datasource_info(
     :raises SupersetException: If the datasource no longer exists
     """
 
-    datasource = form_data.get("datasource", "")
-
-    if "__" in datasource:
+    # pylint: disable=superfluous-parens
```

Comment:
Sadly we're stuck between a rock and a hard place here. There is a Pylint false positive on line #265 but if we add the disable comment there then Black will reformat the line  because its > 88 characters and the formatting then requires the extra `(...)` per Pylint's rules.